core/src/main/java/org/mapfish/print/map/image/wms/WmsUtilities.java

```diff
@@ -69,6 +69,9 @@ public static URI makeWmsGetLayerRequest(final MfClientHttpRequestFactory reques
 
         if (wmsLayerParam.serverType != null && dpi != Constants.PDF_DPI) {
             addDpiParam(extraParams, (int) Math.round(dpi), wmsLayerParam.serverType);
+            int width = (int) (imageSize.width * dpi / Constants.PDF_DPI);
+            int height = (int) (imageSize.height * dpi / Constants.PDF_DPI);
+            getMapRequest.setDimensions(width, height);
```

Comment:
@Jenselme Looking at this again, it seems that your change has not effect. `getMapRequest` is not used anymore after this point. I missed this during the review.